web/test/pages/onboarding/onboarding-owning.test.tsx

```diff
@@ -216,6 +217,7 @@ describe("onboarding - owning a business", () => {
       profileData: generateProfileData({
         businessPersona: "OWNING",
         operatingPhase: "GUEST_MODE_OWNING",
+        legalStructureId: randomLegalStructure().id,
```

Comment:
Regardless of the change to factories, this seems unneeded because oscar also does not select business structure in onboarding anyways